### PR TITLE
Add Andrii Nazaruk folder and push HomeWork

### PR DESCRIPTION
**Pull request by AndriiNazaruk**
Add Andrii Nazaruk folder and push HomeWork

### CODE REVIEW

File: Projects/Team3/Andrii Nazaruk/HomeWork2.2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                summ = firstDigit + secondDigit;
+                Console.WriteLine("FirstDigit = " + firstDigit);
+                Console.WriteLine("FirstDigit = " + secondDigit);
+                Console.WriteLine("Count : " +count);
+                Console.WriteLine("Summ = " +summ);
+            }
+            else
+            {
+                Console.WriteLine("Digit <100");
+            }
```
Comment (romanukhaav) [inline Projects/Team3/Andrii Nazaruk/HomeWork2.2/Program.cs:30]: will your code handle a case when 0< digit <10 ?

File: Projects/Team3/Andrii Nazaruk/HomeWork2.3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+            
+            Console.WriteLine("1.Якщо ви хочете перевести сантиметри в дюйми нажмите 1");
+            Console.WriteLine("2.Якщо ви хочете перевести дюйми в сантиметри нажмите 2");
+            int a = Convert.ToInt32(Console.ReadLine());
+                switch (a)
+                {
+                    case 1:
+                        Console.WriteLine("Введите количество сантиметров");
+                        double b = Convert.ToDouble(Console.ReadLine());
```
Comment (romanukhaav) [inline Projects/Team3/Andrii Nazaruk/HomeWork2.3/Program.cs:22]: Why do you use 2 languages in your menu - Ukrainian and then Russian?

Try to use only English

File: Projects/Team3/Andrii Nazaruk/HomeWork3/Program.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    class Program
+    {
+        
+        //3. Виведіть на екран наступний текст:
+        //    To be or not to be
+        //    \ Shakespeare \
+        static void Main(string[] args)
+        {
+            
+            Console.WriteLine("To be or not to be ");
```
Comment (romanukhaav) [inline Projects/Team3/Andrii Nazaruk/HomeWork3/Program.cs:14]: Can you do this with one command - not with two?

File: Projects/Team3/Andrii Nazaruk/HomeWork4.3/Program.cs [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                {
+                    repeatDigit = array[i];
+                    count++;
+                    Console.WriteLine("Repeat numbers :" + repeatDigit);
+
+                }
+            }
+           
+            Console.WriteLine("Count of repeats : " + count);
+
```
Comment (romanukhaav) [inline Projects/Team3/Andrii Nazaruk/HomeWork4.3/Program.cs:34]: do your program able to count repeats for example in this case : 1,2,3,4,5,1,2,3
and what result will you get?

File: Projects/Team3/Andrii Nazaruk/HomeWork2.2/Program.cs (hunk 1 above)
Comment (AndriiNazaruk) [reply Projects/Team3/Andrii Nazaruk/HomeWork2.2/Program.cs:30]: All right, I will do it again.

File: Projects/Team3/Andrii Nazaruk/HomeWork3/Program.cs (hunk 3 above)
Comment (AndriiNazaruk) [reply Projects/Team3/Andrii Nazaruk/HomeWork3/Program.cs:14]: Just need to add \n to the end of string

File: Projects/Team3/Andrii Nazaruk/HomeWork2.3/Program.cs (hunk 2 above)
Comment (AndriiNazaruk) [reply Projects/Team3/Andrii Nazaruk/HomeWork2.3/Program.cs:22]: Ukrainian is not supported in Microsoft Visual Studioю. All right, I will do it again.

File: Projects/Team3/Andrii Nazaruk/HomeWork3/Program.cs (hunk 3 above)
Comment (Chadoboy) [reply Projects/Team3/Andrii Nazaruk/HomeWork3/Program.cs:14]: Just need to add \n to the end of a string

File: Projects/Team3/Andrii Nazaruk/HomeWork4.3/Program.cs (hunk 4 above)
Comment (AndriiNazaruk) [reply Projects/Team3/Andrii Nazaruk/HomeWork4.3/Program.cs:34]: no, I checked. I will do it again

**Pull request closed without merging**